articles/architecture-scenarios/_includes/_logout/_introduction.md

```diff
@@ -9,8 +9,11 @@ When configuring logout behavior, you'll need to consider:
 * Which sessions should be terminated when the user initiates logout?
 * What information should you provide to users as confirmation of the sessions terminated?
 * Where should users be redirected to after logout completes?
-* Whether you need to account for actions taken by the user elsewhere (such as in another application) to [provide a single logout experience](/logout/guides/logout-applications#single-sign-out-configuration-example)?
 * How long do you want sessions to last in the event that users do not trigger the logout process?
+<% if (platform === "b2b") { %>
+* Should the End User be logged out of all of their application sessions when they log out of one of their applications?
+* Should the session at the organization's IDP also be terminated at logout?
+<% } %>
 
 Given the varying types of sessions that can be created whenever a user logs in, there are several types of logout possible. Local application logout ends the session with the application, whereas Auth0 logout [terminates the Auth0 session](/logout/guides/logout-auth0). [Federated logout](/logout/guides/logout-idps) terminates the Auth0 session and also propagates the logout request to a remote, third-party Identity Provider. Global, or [Single Logout](/logout/guides/logout-applications) (SLO), ends the Auth0 session and also sends a logout request/notice to applications relying on the Auth0 session.
```

Comment:
Hmmm. We talk about federated logout here. Is the additional paragraph below (https://github.com/auth0/docs/pull/8546/files#r356571991) necessary?

Reply:
That's a good question... maybe we should just point this link to the federated logout section...

Reply:
```suggestion
Given the varying types of sessions that can be created whenever a user logs in, there are several types of logout possible. Local application logout ends the session with the application, whereas Auth0 logout [terminates the Auth0 session](/logout/guides/logout-auth0). If you have organizations that are using their own IDP, you may want to consider a [Federated Logout](#federated-logout) strategy and implement accordingly. Global, or [Single Logout](/logout/guides/logout-applications) (SLO), ends the Auth0 session and also sends a logout request/notice to applications relying on the Auth0 session.
```